### ISSUE TITLE
instana.io

### ISSUE THREAD

**Issue by schnism** [lightswitch05/hosts#134]
Hi,

instana is a software performance management solution (https://en.wikipedia.org/wiki/Instana)

It is blocked in ads-and-tracking-extended.txt, however they do not do any tracking or advertising.

Regards,
   Kai

**Comment by lightswitch05** [comment]
Hello @schnism, thank you for taking the time to open this ticket and share your opinion with the project.

**Details about this blocked host:**
* The full URL of why I added `instana.io` is `https://eum.instana.io/eum.min.js`.
* An example website that loads this URL is `https://www.livestrong.com/`.
* The host was originally added in commit: https://github.com/lightswitch05/hosts/commit/4f3d596f
* Company privacy policy: https://docs.instana.io/instana-privacy-policy-latest.pdf

Snippets I found relevant in the privacy policy:

> We also collect location data through the product as to your real time location to provide
location services where requested or agreed to by you in order to deliver content, advertising
or other services that are dependent on knowing where you are. This information may also be
collected in combination with an identifier associated with your device to enable us to
recognize your mobile browser or device when you return to the product website.

> **Who we share your information with**
> Advertisers and advertising networks that require the data to select and serve
relevant adverts to you and others. We do not disclose information about
identifiable individuals to our advertisers, but we will provide them with
aggregate information about our users (for example, we may inform them that 
500 men aged under 30 have clicked on their advertisement on any given day).
We may also use such aggregate information to help advertisers reach the kind
of audience they want to target. We may make use of the personal data we have
collected from you to enable us to comply with our advertisers' wishes by
displaying their advertisement to that target audience and subject to the cookie
section of this policy.

In summary, `instana.io` is fits perfect in this list as a tracker sharing data to enable advertisers and will not be removed. I'll leave this ticket open for a while for others to discuss, but I will not be taking action on this.
